design-document.md

```diff
@@ -210,8 +153,12 @@ K^n.
 
 Prio combines standard [linear secret
 sharing](https://en.wikipedia.org/wiki/Secret_sharing#t_=_n) with a new type of
-probabilistically checkable proof (PCP) system, called a fully linear PCP. The
-input-input validation protocol can be described in terms of three main
+probabilistically checkable proof (PCP) system, called a fully linear PCP. 
+The aggregrators jointly validate the proof of correctness of the input.
```

Comment:
The most recent consensus is that aggregators will decrypt proof+input shares received from clients, forward the proof shares to the leader (while keeping input shares secret), and the leader is responsible for validating the proof and instructing aggregators to include an input in an aggregation.

A further nit: the proof proves that the input is _valid_ (#16 adds a definition of invalid input to the glossary). Prio can't say anything useful about the correctness or truthfulness of the input.